### DISCUSSION TITLE
After inheriting from ScrollViewer, the child items are not rendered in the interface.

### DISCUSSION THREAD

**Discussion by qian-o** [AvaloniaUI/Avalonia#15397] [Q&A] [answered]
Create a class named PreviewScrollViewer that inherits from ScrollViewer. Reference the namespace of PreviewScrollViewer in XAML, and use the control to place display content inside.
![image](https://github.com/AvaloniaUI/Avalonia/assets/84434846/fc1fbdb0-e735-4829-ae1a-d7d65092d2c5)
![image](https://github.com/AvaloniaUI/Avalonia/assets/84434846/55017080-948a-449c-b6cb-3d5387ffb828)
![image](https://github.com/AvaloniaUI/Avalonia/assets/84434846/930c0255-613b-4494-a406-dc79cc449c10)
![image](https://github.com/AvaloniaUI/Avalonia/assets/84434846/03063346-1489-41ff-81b2-2dee71a669b8)

If only using ScrollViewer, there are no issues.
![image](https://github.com/AvaloniaUI/Avalonia/assets/84434846/930a4202-e46b-4348-99ed-b5a2d5e38f3c)

**Comment by timunie** [comment, marked as answer]
You need the Style for the custom control. If it should look like the normal one, use `StyleKeyOverride`
